datafusion/results/f16s_v2.json

```diff
@@ -11,48 +11,48 @@
     "data_size": 14779976446,
 
     "result": [
-[1.365, 0.660, 0.647],
-[0.485, 0.491, 0.460],
-[1.130, 0.992, 0.980],
-[0.864, 0.636, 0.627],
-[2.477, 2.324, 2.317],
-[4.989, 4.707, 4.687],
-[0.470, 0.422, 0.421],
-[0.529, 0.510, 0.493],
-[3.175, 2.934, 2.886],
-[5.496, 5.113, 5.056],
-[2.001, 1.715, 1.711],
-[2.524, 2.162, 2.156],
-[5.354, 5.132, 5.221],
-[7.277, 7.016, 6.849],
-[6.135, 5.724, 5.722],
-[4.286, 4.023, 4.126],
-[10.519, 10.005, 9.987],
-[8.287, 7.855, 7.898],
-[17.448, 17.179, 16.774],
-[0.753, 0.568, 0.566],
-[26.014, 10.427, 10.361],
-[15.792, 13.861, 13.542],
-[52.167, 37.184, 36.419],
-[158.124, 105.186, 104.726],
-[5.948, 5.328, 5.312],
-[7.518, 7.221, 7.207],
-[9.740, 9.051, 9.320],
-[12.577, 10.655, 10.695],
-[248.741, 253.971, 255.908],
-[2.263, 2.674, 2.652],
-[8.128, 7.429, 7.305],
-[10.015, 8.755, 8.773],
+[1.397, 0.683, 0.653],
+[0.323, 0.245, 0.273],
+[0.979, 0.725, 0.750],
+[1.003, 0.662, 0.646],
+[2.709, 2.491, 2.502],
+[5.380, 4.981, 4.912],
+[0.358, 0.221, 0.217],
+[0.310, 0.263, 0.246],
+[3.360, 3.095, 2.981],
+[52.658, 52.928, 52.789],
```

Comment:
are you sure everything was ok with this query? now it is 10x worse.

Reply:
thank you @nickitat , will generate index.html next time.

we found the performance drop as well. it might be caused by the new implementation of distinct function. tracked by https://github.com/apache/arrow-datafusion/issues/5313 now.